test/integration-test.py

```diff
@@ -168,19 +169,32 @@ def verify_ct_submission(expectedSubmissions, url):
         die(ExitStatus.CTFailure)
     return 0
 
-def run_node_test(domain, chall_type, expected_ct_submissions):
+def run_node_test(domain, chall_type, expected_ct_submissions, abort_step="null"):
     email_addr = "js.integration.test@letsencrypt.org"
     cert_file = os.path.join(tempdir, "cert.der")
     cert_file_pem = os.path.join(tempdir, "cert.pem")
     key_file = os.path.join(tempdir, "key.pem")
+
+    if abort_step is None or abort_step == "":
```

Comment:
This is weird - the function declaration has a default for the abort_step value but we also have one here. We should pick one and stick to it.